app/css/ui-kit/colors.css

```diff
@@ -240,6 +240,20 @@
     @apply text-textColor5 bg-backgroundColorB;
 }
 
+@layer base {
+    .alert-filter {
+        filter: var(--filter-D85050);
+    }
+
+    .textColor6-filter {
+        filter: var(--textColor6-filter);
+    }
+
+    .white-filter {
+        filter: var(--white-filter);
+    }
+}
```

Comment:
Can you put these in `app/css/ui-kit/filters.css` pls, and switch the name so it's `filter-xxx`